en/services/opendroneid.md

```diff
@@ -46,27 +51,53 @@ Basic ID | Basic ID | [OPEN_DRONE_ID_BASIC_ID](../messages/common.md#OPEN_DRONE_
+- | - | [OPEN_DRONE_ID_SYSTEM_UPDATE](../messages/common.md#OPEN_DRONE_ID_SYSTEM_UPDATE) | A subset of the [OPEN_DRONE_ID_SYSTEM](#OPEN_DRONE_ID_SYSTEM) message, containing only fields that must be updated regularly by the GCS. Can optionally be used instead of the full `OPEN_DRONE_ID_SYSTEM` message, to reduce the traffic on the control link.
+
 
-> **Note** The raw byte layout of the MAVLink messages is not exactly the same as what a drone ID Bluetooth/Wi-Fi transmitter will transmit over the air.
+> **Note** The raw byte layout of the MAVLink messages is not exactly the same as what a RID Bluetooth/Wi-Fi transmitter component will broadcast over the air.
   Slight compression is applied.
   Example code for this conversion can be found in the project: [Open Drone ID Core C Library](https://github.com/opendroneid/opendroneid-core-c).
 
-The [Open Drone ID Core C Library](https://github.com/opendroneid/opendroneid-core-c) contains code for decoding the MAVLink messages and "compressing" the data into data structures for transmission over Bluetooth or Wi-Fi (or vice-versa).
-
-The standards requires that the Location message is broadcast/published at least once per second.
-The rest of the messages must be broadcast/published once per 3 seconds (requirements from local legislation might be different.
-Please see [here](https://github.com/opendroneid/opendroneid-core-c#comparison)).
-Not all message types are mandatory to broadcast.
+The [Open Drone ID Core C Library](https://github.com/opendroneid/opendroneid-core-c) contains code for decoding the MAVLink messages and "compressing" the data into data structures for broadcast over Bluetooth or Wi-Fi (or vice-versa for reception).
 
 The standards do not impose any requirements for a drone to be capable of receiving drone ID messages, nor any requirements for reacting to their content (requirements from local legislation might be different).
 
-An example Android receiver implementation for broadcast drone ID messages is available here: [OpenDroneID Android receiver application](https://github.com/opendroneid/receiver-android).
+An example Android receiver implementation for broadcast drone ID messages is [available here](https://play.google.com/store/apps/details?id=org.opendroneid.android_osm) ([source codes](https://github.com/opendroneid/receiver-android)).
 
 Code related to (Internet) Network Remote ID can be found in the [InterUSS Project](https://github.com/interuss) and https://github.com/uastech/standards (Unofficial reference for UAS-related APIs).
 
+## Message update rates {#update_rates}
+
+The standards require that the Location message is broadcast/published from the RID transmitter component at least once per second.
```

Comment:
"The standards" us quite broad and might change - say Australia adds a standard. We should at least link to which ones we mean here and now. I presume we mean the stuff in specifications?

```suggestion
ASTM F3411 and ASD-STAN prEN 4709-00 (and possibly other [standards ](#specifications)) require that the Location message is broadcast/published from the RID transmitter component at least once per second.
```

Reply:
Added